content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -4185,6 +4185,547 @@ Timestamp,Name,Login,Event,Description,SourceIP,RequireOrgAdmin,RequireStackAdmi
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  https://api.pulumi.com/api/orgs/{organization}/agent-pools/{poolId}
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "created": 1715701863000,
+  "id": "12345678-8102-447f-b246-e9ec85786e23",
+  "name":"Production",
+  "description": "Pool for the production account",
+  "agents": [
+    {
+      "version": "v1.1.0",
+      "hostname": "private-ipv4-address.kind.internal",
+      "ip": "192.168.0.17",
+      "pid": "58188",
+      "lastSeen": 1719498194000,
+      "status": "online"
+    }
+  ]
+}
+```
+
+### List registered pools
+
+```
+GET /api/orgs/{organization}/agent-pools
+```
+
+#### Parameters
+
+| Parameter           | Type      | In    | Description         |
+|---------------------|-----------|-------|---------------------|
+| `organization`      | string    | path  | organization name   |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  https://api.pulumi.com/api/orgs/{organization}/agent-pools
+```
+
+#### Default response
+
+```
+Status: 200 OK
+```
+
+```
+{
+  "agentPools": [
+    {
+      "created": 1715701863000,
+      "id": "12345678-8102-447f-b246-e9ec85786e23",
+      "name":"Production",
+      "description": "Pool for the production account",
+      "lastSeen": 1715796999,
+      "status": "online",
+      "lastDeployment": 1715796961
+    }
+  ]
+}
+```
+
+## OIDC Issuers
+
+<!-- ###################################################################### -->
+
+### Register a new issuer
+
+```
+POST /api/orgs/{organization}/oidc/issuers
+```
+
+#### Parameters
+
+| Parameter           | Type          | In    | Description         |
+|---------------------|---------------|-------|---------------------|
+| `organization`      | string        | path  | organization name   |
+| `name`              | string        | body  | oidc issuer name    |
+| `url`               | string        | body  | issuer base url (this will be used as a base to build the OIDC configuration url, `url + /.well-known/openid-configuration`) |
+| `thumbprints`       | array[string] | body  | **Optional.** issuer TLS certificate thumbprints |
+| `maxExpiration`     | int           | body  | **Optional.** max expiration for tokens issued for this issuer in seconds    |
+| `jwks`              | json ([jwks format](https://datatracker.ietf.org/doc/html/rfc7517)) | body  | **Optional.** JWK Set from the OIDC issuer    |
+
+#### Example
+
+```bash
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{ "name":"GitHub", "description": "https://token.actions.githubusercontent.com", "maxExpiration": 3600 }' \
```

Comment:
Should `"description"` be `"url"` instead?

Reply:
yes!

Reply:
done